ui/DesignSystem/Component/KeyHint.svelte

```diff
@@ -0,0 +1,33 @@
+<script>
+  import { isMac } from "./../../src/settings.ts";
+
+  export let hint = null;
+  export let noModifier = false;
+  export let style = null;
+
+  const modifierKey = isMac ? "⌘" : "ctrl";
+</script>
+
+<style>
+  .hint-container {
+    display: flex;
+  }
+
+  .hint {
+    background-color: var(--color-foreground-level-2);
+    color: var(--color-foreground-level-5);
+    padding: 0.125rem 0.375rem;
+    border-radius: 0.25rem;
+  }
+
+  .hint:first-child {
+    margin-right: 0.25rem;
+  }
+</style>
+
+<div class="hint-container" {style}>
+  {#if hint.length === 1 && !noModifier && hint !== '↵'}
```

Comment:
I would probably have this validation in a variable:
```typescript
    const addModifier = hint.length === 1 && !noModifier && hint !== '↵`;
```

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/914/commits/4da650c03cb5846cdd55d435f473cb6dde0749fd